config/database.yml

```diff
@@ -17,6 +17,10 @@
 default: &default
   adapter: postgresql
   encoding: unicode
+  username: postgres
+  password: 1234
+  host: localhost
+  port: 5432
```

Comment:
### OPTIONAL
- [ ] It is always good practice not to expose your database login information. Consider using the env file for all sensitive information about your application.